p4broker/plan.sh

```diff
@@ -0,0 +1,39 @@
+pkg_name=p4broker
+pkg_origin=core
+pkg_version="18.1"
+pkg_maintainer="Daniel Hagen <daniel.b.hagen@gmail.com>"
```

Comment:
All core plans should have the maintainer set to `The Habitat Maintainers <humans@habitat.sh>`